src/browserContextFactory.ts

```diff
@@ -229,3 +238,18 @@ async function findFreePort(): Promise<number> {
     server.on('error', reject);
   });
 }
+
+function cacheDirectory(): string {
```

Comment:
Use `registryDirectory` from 'playwright-core/lib/server/registry/index' instead of reimplementing it here.